### ISSUE TITLE
Scaling on Linux

### ISSUE THREAD

**Issue by tuguldurio** [sindresorhus/wallpaper#55]
on linux when setting wallpaper using `wallpaper.set(path, {scale: blabla})` scale option seems to be not working.

**Comment by sindresorhus** [comment]
From the docs:

> scale (macOS only)

Notice the “macOS” part.

**Comment by tuguldurio** [comment]
so for now there is no possibility right?

**Comment by sindresorhus** [comment]
Correct. I’ll keep this open as a feature request, but it’s not something I’m gonna work on.

**Comment by beyluta** [comment]
> so for now there is no possibility right? edit: is there any possibility could it implemented in linux and windows?

I am working on implementing this feature on windows. Some changes to the binary were required since it did not allow for this feature as it was. I sent a PR to the win-wallpaper repo. As soon as it gets merged, I will work on updating this repository as well.

**Comment by sindresorhus** [comment]
Correct, and this is by design. `scale` is documented as macOS and Windows only, and `source/linux/index.js` never accepted the option at all.

Making it work is not a small fix. The 15 Linux backends have non-overlapping vocabularies (`picture-options`, `--bg-fill`, `--set-zoom-fill`, `--resize`, and others), and the design runs every manager that is available, so there is no defined rule for which one wins. I am not adding desktop detection for that. Added one line to the readme to make this explicit.
